packages/inter-protocol/src/vaultFactory/liquidation.js

```diff
@@ -53,16 +84,36 @@ const scheduleLiquidationWakeups = async (
   const auctionStartDelay = params[AUCTION_START_DELAY].value;
 
   const nominalStart = TimeMath.subtractAbsRel(startTime, auctionStartDelay);
+  const afterStart = TimeMath.addAbsRel(startTime, 1n);
+
+  // scheduleLiquidationWakeups is supposed to be called just after an auction
+  // started. If we're late but there's still time for the nominal start, then
+  // we'll proceed. Reschedule for the next round if that's still in the future.
+  // Otherwise, wait for governance params to change.
+  if (TimeMath.compareAbs(now, nominalStart) > 0) {
+    if (TimeMath.compareAbs(now, endTime) < 0) {
+      if (cancelToken) {
+        E(timer).cancel(cancelToken);
+      }
+
+      void E(timer).setWakeup(afterStart, reschedulerWaker);
+      return;
+    } else {
+      waitForNewAuctionParams();
+      return;
+    }
+  }
+
+  cancelToken = cancelToken || harden({});
```

Comment:
```suggestion
  cancelToken ||= makeCancelToken();
```

Reply:
`||=` was introduced by ES2021. We have had bad experience with some tools in our toolchain not understanding newer syntax. I don't know what the cutoff date is for such worries. But if no one else knows better, I would recommend not making this change.

Reply:
I thought I fixed that with https://github.com/endojs/endo/pull/1520. But yeah there may be other tools and today's not the day to find out.